lib/src/base-styles/_nx-globals.scss

```diff
@@ -47,3 +47,14 @@ $-nx-scrollable-default-height: 400px;
 .nx-blockquote {
   @extend %nx-blockquote;
 }
+
+//Class that allows making content visible only to screen readers but not visually
+.nx-visually-hidden {
```

Comment:
> I'm interested in what the rest of the team thinks, but I would tend to prefer applying the relevant styles to `.nx-cell--header.nx-cell--row-btn` or the appropriate child thereof, rather than making this utility class. I try to avoid making classes that are named after their visual effect, and prefer classes that designate ad-hoc semantics of what they're applied to.

Changed here: 561e7b33df46898d6c5f37a2adcf7f74d79ab0a8